xarray/core/missing.py

```diff
@@ -10,6 +10,7 @@
 import numpy as np
 import pandas as pd
 
+import xarray as xr
```

Comment:
If you look at how other imports from xarray are handled in this file you will see that we don't use `import xarray as xr` internally, instead preferring "explicit imports", e.g. `from xarray.core.indexes import IndexVariable`